packaging/faucet/miden-faucet.service

```diff
@@ -7,8 +7,8 @@ WantedBy=multi-user.target
 
 [Service]
 Type=exec
-Environment="RUST_LOG=info"
-ExecStart=/usr/bin/miden-faucet start --config /etc/opt/miden-faucet/miden-faucet.toml
+EnvironmentFile=/lib/systemd/system/miden-faucet.env
```

Comment:
@Mirko-von-Leipzig regarding override -

Should be as simple as editing `/lib/systemd/system/miden-faucet.env` and restarting service. No need for override file. Would expect to use an override file to change things specific to systemd rather than the faucet app e.g. these settings

```
RestartSec=5
Restart=always
```

Reply:
Right but how do you do this programmatically from a deployment perspective?

I think my expectation would be that the package is installed with sensible defaults, and now I want to specify some of these settings as part of my deployment.

1. I install the the deb package,
2. Now how do I specify my settings?

As a general statement, the node already comes with defaults as part of the CLI - do we want/need additional env configuration that is part of the debian package that repeats those?

Reply:
Seems a comment of mine in this thread also got lost 😭 

>As a general statement, the node already comes with defaults as part of the CLI - do we want/need additional env configuration that is part of the debian package that repeats those?

So we could do away with the `.env` and as you say
1. Rely on defaults from the source;
2. Leave users / automation to specify flags or env vars manually for Dockerfile; and
3. Use the systemd override approach for Debian packages.

I do feel like the `.env` file helps out point 2 but probably not enough to outweigh its removal?

Reply:
> I still feel uneasy about this. We're expecting this directory to exist and whoever installs this package will now need to go edit this additional file they don't really know about. Basically I feel like the default package install shouldn't come with an additional env file. It should just set the values which don't have defaults in the service file.

Yea this sounds like a fair criticism. Lets keep it top of mind and come back to it if required.